[b2c-support]: Update threat-management.md (#75428)
Azure/azure-content: articles/active-directory-b2c/threat-management.md

## Unlock accounts

> When a user enters correct password immediately after the account is locked out, it does not take lockout duration into consideration and allows the user to sign-in successfully. However, if the user keeps entering wrong password, the lockout duration increases. The first 10 lockout periods are one minute long. The next 10 lockout periods are slightly longer and increase in duration after every 10 lockout periods. The lockout counter resets to zero after a successful login when the account isn’t locked. Lockout periods can last up to five hours. Users must wait for the lockout duration to expire. However, the user can unlock by using self-service [password user flow](add-password-reset-policy.md).

Reviewer: Hi @rohsh354, just a suggestion to help clarify. Is this accurate? 

```suggestion
If a user enters the correct password immediately after their account is locked, the lockout duration isn't affected and the user is allowed to sign in successfully. However, if the user keeps entering the wrong password, the lockout duration increases. The first 10 lockout periods are one minute long. The next 10 lockout periods are slightly longer and increase in duration after every 10 lockout periods. The lockout counter resets to zero after a successful login when the account isn’t locked. Lockout periods can last up to five hours. Users must wait for the lockout duration to expire. However, the user can unlock by using self-service [password user flow](add-password-reset-policy.md).
```
